File: event/src/test/java/de/danoeh/antennapod/event/DownloadLogEventTest.java
package de.danoeh.antennapod.event;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotSame;

public class DownloadLogEventTest {

    @Test
    public void listUpdatedReturnsNewInstanceEachCall() {
        DownloadLogEvent first = DownloadLogEvent.listUpdated();
        DownloadLogEvent second = DownloadLogEvent.listUpdated();
        assertNotSame(first, second);
    }

    @Test
    public void toStringIsFixedString() {
        assertEquals("DownloadLogEvent", DownloadLogEvent.listUpdated().toString());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/DownloadLogEventTest.kt
package de.danoeh.antennapod.event

import org.junit.Assert.assertEquals
import org.junit.Assert.assertNotSame
import org.junit.Test

class DownloadLogEventTest {

    @Test
    fun listUpdatedReturnsNewInstanceEachCall() {
        val first = DownloadLogEvent.listUpdated()
        val second = DownloadLogEvent.listUpdated()
        assertNotSame(first, second)
    }

    @Test
    fun toStringIsFixedString() {
        assertEquals("DownloadLogEvent", DownloadLogEvent.listUpdated().toString())
    }
}
